### ISSUE TITLE
Emoji Selector in Editors

### ISSUE THREAD

**Issue by thamaraiselvam** [Hashnode/user-feedback#78]
**Is your feature request related to a problem? Please describe.**
I express my self as with emojis most of the time but there is no emoji way to include emojis on Comments text box

**Describe the solution you'd like**
Bring emojis selector on all editors such as Story and comments.

**Describe alternatives you've considered**
Now emojis are able to copy and paste from source but that consumes lots of time

**Comment by minecrawler** [comment]
Your OS offers an emoji input, so I don't really think this is necessary. However, it would be cool to have instructions on how to use it, because I think most people don't know. For *NIX, maybe a fallback with the 10 most common emoji would be cool.

- **Windows**: `[WINDOWS]` + `[.]`
- **macOS**: `[COMMAND]` + `[CONTROL]` + `[SPACE]`

Other *NIX are a bit more diverse, and may require additional packages. They usually seem to work with `[CONTROL]` + `[.]`, though, but can be rebound to other combinations, of course.

**Comment by thamaraiselvam** [comment]
Makes sense but I'm using Windows 7 (MNC Licenses ..blah blah) it does not support it by default. I had to copy and paste it somewhere.

**Comment by minecrawler** [comment]
Is it implemented, will there be work on it, or is it rejected, because Win7 is not supported anymore anyway?

**Comment by BolajiAyodeji** [comment]
Hi @minecrawler, apologies as I closed this by mistake.

This is great feedback and I'll relate this to my team and provide some update.

I'll be re-opening for now.
